topics/sequence-analysis/tutorials/ref-based-rad-seq/tutorial.md

```diff
@@ -39,6 +39,10 @@ You can directly used archive from the Sequence Read Archive (SRA) for raw reads
 
 To download all training datasets (i.e reads, reference genome, population map file and barcodes file), you need to use the corresponding [CeSGO hub](https://cesgo.genouest.org/resources/370/supportingdocs) repository.
 
+TODO put on zenodo? get SRA data with SRA tool? 
+TODO chromFa.tar.gz upload does not work, Galaxy gunzips the file leaving a tar that leads to a problem in BWA
```

Comment:
we could upload a chromFa.gz to zenodo I guess.

Reply:
I have uplaoded a ~230Mb fasta genome file. Is it ok ? Concerning the fastq file, Is it better to put it on the Zenodo repo or let the user download form SRA ? I'm more for the second option but maybe this can be interesting to have all material in the zenodo place ?

Reply:
Imho, SRA is ok and people should learn to deal with SRA/ENA. So fastq I would leave on SRA.

Reply:
Here is the Zenodo repo https://zenodo.org/record/1134547#.WkugUjfjJXI